### DISCUSSION TITLE
Trying to add a remote repository to my git-bash terminal

### DISCUSSION THREAD

**Discussion by OceanMan11235** [community/community#45702] [New to GitHub]
I am following Github's guides, and am now following along with Managing remote repositories. However, when I try to follow their steps by typing in:
"$ git remote add origin https://github.com/OceanMan11235/First-Collab.git"
it says:
"fatal: not a git repository (or any of the parent directories): .git"

**Comment by airtower-luna** [comment]
> "fatal: not a git repository (or any of the parent directories): .git"

That means your current working directory isn't part of a Git repository. You need to change into the directory where your repository is (using `cd`).

If you want to know what your current working directory is, use `pwd`.

**Comment by shree675** [comment]
@OceanMan11235 If you are in the right local directory, you have to run the below command first to initialize your local directory as a git repository:
```powershell
git init
```
Then you can add a remote branch by following the guide.
